runtime/src/bank.rs

```diff
@@ -1108,9 +1109,18 @@ impl Bank {
 
         let (epoch_stakes, epoch_stakes_time_us) = measure_us!(parent.epoch_stakes.clone());
 
-        let (transaction_processor, builtin_program_ids_time_us) = measure_us!(
-            TransactionBatchProcessor::new_from(&parent.transaction_processor, slot, epoch)
-        );
+        let (transaction_processor, new_transaction_processor_timings) =
+            TransactionBatchProcessor::new_from(
+                &parent.transaction_processor,
+                parent.as_ref(),
```

Comment:
The accounts were loaded from `new` not from `parent`.

Reply:
Ahh, I see the comment about touching AccountsDB in the epoch update step. Good call, likely better to use `new`. Done!